DESCRIPTION

```diff
@@ -18,4 +18,5 @@ Depends:
     rAzureBatch (>= 0.4.0)
 Suggests:
     testthat, caret, plyr, lintr
+Remotes: Azure/rAzureBatch
```

Comment:
Very cool! I was not familiar with this.

Does this mean we should update our docs & samples to remove the reference to Azure/rAzureBatch as a required install?